skimage/color/tests/test_colorconv.py

```diff
@@ -729,3 +730,27 @@ def test_gray2rgba_alpha():
     with pytest.raises(ValueError) as err:
         rgba = gray2rgba(img, alpha)
     assert expected_err_msg == str(err.value)
+
+
+@pytest.mark.parametrize("func", [rgb2hsv, hsv2rgb,
```

Comment:
how many functions have a different shape for input and output? I can see `rgb2gray` for sure but maybe others. Splitting in two test functions, one for functions preserving the shape and another for the other functions, would allow us to have a more coercive test for functions preserving the shape.

Reply:
I think `rgb2gray` is the only one!

Reply:
`gray2rgb`, `gray2rgba`, `rgb2gray` and `rgba2rgb` are the four functions that have a different input and output shape. Should I write specific tests for these functions? Because only `rgba2rgb` can't fit this test because input shape last dimension must be 4 instead of 3...

Reply:
Ideally - but I know I'm nitpicking here - I would keep the tests which you have written for `gray2rgb`, `gray2rgba`, and `rgb2gray`, write a specific test for `rgba2rgb` and for all functions which preserve shape, test that the output shape is exactly the input shape.

Reply:
I implemented your suggestion, thank you @emmanuelle